views/container-statuses.html

```diff
@@ -0,0 +1,17 @@
+<div ng-if="containerStatuses.length == 0"><em>none</em></div>
+<dl ng-repeat="containerStatus in containerStatuses | orderBy:'name'" class="dl-horizontal">
```

Comment:
does this look weird if there are multiple container statuses? not sure if there's any separation between them... maybe add a container status for each state we render to the fixture